core/server/src/main/java/tachyon/master/file/options/SetAttributeOptions.java

```diff
@@ -0,0 +1,161 @@
+/*
+ * Licensed to the University of California, Berkeley under one or more contributor license
+ * agreements. See the NOTICE file distributed with this work for additional information regarding
+ * copyright ownership. The ASF licenses this file to You under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License. You may obtain a
+ * copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package tachyon.master.file.options;
+
+import javax.annotation.concurrent.NotThreadSafe;
+
+import tachyon.conf.TachyonConf;
+import tachyon.master.MasterContext;
+import tachyon.thrift.SetAttributeTOptions;
+
+/**
+ * Method option for setting the attributes.
+ */
+@NotThreadSafe
+public class SetAttributeOptions {
```

Comment:
yes, I would like to change all the server-side options to follow the client-side style in a follow up PR